frontend/src/app/containers/App/index.js

```diff
@@ -226,6 +226,12 @@ class App extends Component {
         if (typeof localizations[lang] === "string") {
           const cx = new MessageContext(lang, {useIsolating: false});
           cx.addMessages(localizations[lang]);
+
+          // HACK: #3631 - host the attribute for this string up into its own separate string
+          const placeholderHackString = cx._messages
+            .get("newsletterFormEmailPlaceholder").attrs.placeholder;
+          cx.addMessages(`newsletterFormEmailPlaceholderHackedString = ${placeholderHackString}`);
```

Comment:
Accessing `_messages` is risky here, because the internal API is subject to change. Instead, you could construct a new message which directly references the attribute (on the Fluent side). Note that that's a regular string, not a template literal, and the `{}` are Fluent syntax.

```js
cx.addMessages("newsletterFormEmailPlaceholderHackedString = {newsletterFormEmailPlaceholder.placeholder}");
```

In order to make it less hacky still, it might be a good idea to add that new message to `app.ftl` to avoid having to construct it programmatically on the runtime. Maybe in a separate file which isn't exposed to localizers?

```properties
newsletterFormEmailPlaceholderHackedString = {newsletterFormEmailPlaceholder.placeholder}
```

@flodolo, what's your take on adding such message to Test Pilot's FTL?

Reply:
FWIW, I'd like to make this slightly less hacky, but not so much less hacky that we're comfortable with forgetting it's there and leaving things this way. Ultimately, this hack should be stable until it can be replaced with something cleaner after a fluent upgrade

Reply:
I wonder if it makes sense to just wait on this til the new fluent release mentioned in https://github.com/mozilla/testpilot/issues/3631#issuecomment-411419408

Reply:
Well, the reason I threw this together is that this error breaks the site for me right now. Without this fix, I can't actually view the site on my machine. I had to have this in the background while getting through to diagnosing other errors yesterday

Reply:
> FWIW, I'd like to make this slightly less hacky, but not so much less hacky that we're comfortable with forgetting it's there and leaving things this way.

Makes sense to me. In this case I recommend the slightly less-ugly approach of

```js
cx.addMessages("newsletterFormEmailPlaceholderHackedString = {newsletterFormEmailPlaceholder.placeholder}");
```

:)